content/docs/tasks/telemetry/export-logs-through-stackdriver.md

```diff
@@ -0,0 +1,259 @@
+                referer: referer
+              labelNames:
+              - sourceIp
+              - destinationIp
+              - sourceService
+              - sourceUser
+              - sourceNamespace
+              - destinationIp
+              - destinationService
+              - destinationNamespace
+              - apiClaims
+              - apiKey
+              - protocol
+              - method
+              - url
+              - responseCode
+              - responseSize
+              - requestSize
+              - latency
+              - connectionMtls
+              - userAgent
+              - responseTimestamp
+              - receivedBytes
+              - sentBytes
+              - referer
+              sinkInfo:
+                id: '<sink_id>'
+                destination: '<sink_destination>'
+                filter: '<log_filter>'
+        ---
+        apiVersion: "config.istio.io/v1alpha2"
+        kind: rule
+        metadata:
+          name: stackdriver
+          namespace: istio-system
+        spec:
+          match: "true" # If omitted match is true.
+          actions:
+          - handler: handler.stackdriver
+            instances:
+            - accesslog.logentry
+        ---
+    ```
+
+1.  Push the configuration
+
+    ```command
+    $ kubectl apply -f stackdriver.yaml
+    stackdriver "handler" created
+    rule "stackdriver" created
+    logentry "stackdriverglobalmr" created
+    metric "stackdriverrequestcount" created
+    metric "stackdriverrequestduration" created
+    metric "stackdriverrequestsize" created
+    metric "stackdriverresponsesize" created
+    ```
+
+1.  Send traffic to the sample application.
+
+    For the Bookinfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+    browser or issue the following command:
+
+    ```command
+    $ curl http://$GATEWAY_URL/productpage
+    ```
+
+1.  Verify that logs are flowing through Stackdriver to the configured sink.
+
+    *   Stackdriver: Navigate to [Stackdriver Logs
+        Viewer](https://pantheon.corp.google.com/logs/viewer) for your project
+        and look under "GKE Container" -> "Cluster Name" -> "Namespace Id" for
+        Istio Access logs.
+    *   BigQuery: Navigate to [BigQuery
+        Interface](https://bigquery.cloud.google.com/) for your project and you
+        should find a table with prefix `accesslog_logentry_istio` in your sink
+        dataset.
+    *   GCS: Navigate to [Storage
+        Browser](https://pantheon.corp.google.com/storage/browser/) for your
+        project and you should find a bucket named
+        `accesslog.logentry.istio-system` in your sink bucket.
+    *   Pub/Sub: Navigate to [Pub/Sub
+        TopicList](https://pantheon.corp.google.com/cloudpubsub/topicList) for
+        your project and you should find a topic for `accesslog` in your sink
+        topic.
+
+## Understanding what happened
+
+`Stackdriver.yaml` file above configured Istio to send metric and accesslog to
```

Comment:
metric -> metrics

Reply:
Removed.. thanks realized that we edited stackdriver.yaml file above just for logs